src/help.ts

```diff
@@ -113,6 +125,15 @@ export const usage: Action = ({ path, args: [command], error, help }) => {
     return 0;
   }
 
+  if (command === "") {
+    error(`Expected a command, but the value was empty\n\n${
+      help({
+        description: false,
+      })
+    }`);
```

Comment:
This feels unnecessarily long-winded. Couldn't I just do:
```suggestion
    error(`Expected a command, but the value was empty`, {
      description: false,
    });
```
?